exchanges/ftx/ftx_test.go

```diff
@@ -91,11 +91,17 @@ func TestGetOrderbook(t *testing.T) {
 
 func TestGetTrades(t *testing.T) {
 	t.Parallel()
-	_, err := f.GetTrades(spotPair, 0, 0, 200)
+	// test empty market
+	_, err := f.GetTrades("", 0, 0, 200)
+	if err == nil {
+		t.Error(err)
```

Comment:
err --> "error cannot be nil"

Reply:
There were quite a lot of these, they should be fine now

Reply:
It's unrelated to FTX, but there are two more instances after I ctrl+f'd, one in btcmarkets_test.go:247 and another in bitfinex_test.go:405 if you feel like completely eradicating this scourge.